resources/execute.rb

```diff
@@ -13,6 +13,7 @@
 attribute :cwd, kind_of: String
 attribute :environment, kind_of: Hash, default: Hash.new
 attribute :group, kind_of: [String, Integer]
+attribute :live_stream, kind_of: Boolean, default: false
```

Comment:
Darn, I completely overlooked this one. For some reason `Boolean` is not available as attribute type and we have to use `[TrueClass, FalseClass]` here.